<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialoghistorique</class>
 <widget class="QDialog" name="Dialoghistorique">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1227</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des hitoriques</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>340</y>
     <width>351</width>
     <height>201</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #8B0000;
  font-size: 16px;</string>
   </property>
   <property name="title">
    <string>Modifier Ou Supprimer un historique</string>
   </property>
   <widget class="QPushButton" name="pushButtonModifier">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #8B0000	;
  border: none;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 11px;
  margin: 4px 2px;
  border-radius : 16px;</string>
    </property>
    <property name="text">
     <string>Modifier</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSupprimer">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #8B0000	;
  border: none;
  color: white;
  padding: 10px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 11px;
  margin: 4px 2px;
  border-radius : 16px;</string>
    </property>
    <property name="text">
     <string>Supprimer</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>311</width>
     <height>501</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #8B0000;
  font-size: 16px;</string>
   </property>
   <property name="title">
    <string>Ajouter un Historique</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  border: none;
  color: black;
  
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 20px;
  </string>
    </property>
    <property name="text">
     <string>ID_H</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  border: none;
  color: black;
  
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 20px;
  
</string>
    </property>
    <property name="text">
     <string>Congé debut</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  border: none;
  color: black;
  
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 20px;
  </string>
    </property>
    <property name="text">
     <string>Pésence</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  border: none;
  color: black;
  
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 20px;
  
</string>
    </property>
    <property name="text">
     <string>Congé fin</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LeId">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAjouter">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>430</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #8B0000	;
  border: none;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 16px;
  margin: 4px 2px;
  border-radius : 16px;</string>
    </property>
    <property name="text">
     <string>Ajouter </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  border: none;
  color: black;
  
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 20px;
  </string>
    </property>
    <property name="text">
     <string>Tache réalisé</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>210</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  border: none;
  color: black;
  
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 20px;
  
</string>
    </property>
    <property name="text">
     <string>Cause</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>370</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>260</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>320</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>150</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;</string>
    </property>
    <item>
     <property name="text">
      <string>Oui</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Non</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  border: none;
  color: black;
  
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 20px;
  </string>
    </property>
    <property name="text">
     <string>ID_Emp</string>
    </property>
   </widget>
   <widget class="QComboBox" name="IDE">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>340</y>
     <width>481</width>
     <height>201</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #8B0000;
  font-size: 16px;</string>
   </property>
   <property name="title">
    <string>Rechercher ou Trier les historiques</string>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  border: none;
  color: black;
  
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 20px;
  
</string>
    </property>
    <property name="text">
     <string>Rechercher par ID_H</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  border: none;
  color: black;
  
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 20px;
  
</string>
    </property>
    <property name="text">
     <string>Rechercher par Présence</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LeIdRT">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LeTitreRT">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>70</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #8B0000	;
  border: none;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 11px;
  margin: 4px 2px;
  border-radius : 16px;</string>
    </property>
    <property name="text">
     <string>Trier par id_h</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>150</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #8B0000	;
  border: none;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 11px;
  margin: 4px 2px;
  border-radius : 16px;</string>
    </property>
    <property name="text">
     <string>Trier par présence</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  border: none;
  color: black;
  
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 20px;
  
</string>
    </property>
    <property name="text">
     <string>Rechercher par ID_Emp</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>110</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>150</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #8B0000	;
  border: none;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 11px;
  margin: 4px 2px;
  border-radius : 16px;</string>
    </property>
    <property name="text">
     <string>Trier par id_emp</string>
    </property>
   </widget>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>831</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black;
border-radius: 10px;
padding:  0 9px;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>7</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #8B0000	;
  border: none;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 11px;
  margin: 4px 2px;
  border-radius : 16px;</string>
   </property>
   <property name="text">
    <string>Retour</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>-40</x>
     <y>-90</y>
     <width>1271</width>
     <height>651</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <zorder>label_4</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox</zorder>
  <zorder>groupBox_3</zorder>
  <zorder>tableView</zorder>
  <zorder>pushButton_4</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
